arviz/data/io_cmdstanpy.py

```diff
@@ -527,29 +527,31 @@ def _unpack_fit(fit, items, save_warmup):
         else:
             num_warmup = fit.num_draws_warmup
 
+    nchains = fit.chains
     draws = np.swapaxes(fit.draws(inc_warmup=save_warmup), 0, 1)
     sample = {}
     sample_warmup = {}
 
     for item in items:
         if item in fit.stan_vars_cols:
             col_idxs = fit.stan_vars_cols[item]
+            if len(col_idxs) == 1:
+                raw_draws = draws[..., col_idxs[0]]
```

Comment:
I figured it would be better to use `draws` with scalar variables to avoid unnecessary reshapes and swapaxes.